### ISSUE TITLE
CEP Incorreto

### ISSUE THREAD

**Issue by gucalves** [pedro-teixeira/correios#446]
Olá, estou tendo problemas no cálculo do frete com alguns ceps, como exemplo o cep 75400-000, ele calcula normalmente na loja e permite que o cliente finalize a compra, quando enviamos a etiqueta com o rastreio para os Correios eles pedem para confirmar o endereço e reetiquetar porque o cep não bate com a base de ceps dos Correios, etc.... o cep correto informando posteriormente pelo cliente seria o 75400-864, mas ele não deveria permitir o cálculo do cep 74500-000, correto?

**Comment by rafaelpatro** [comment]
Concordo @gucalves ! O webservice não deveria retornar cotação para CEPs inválidos.
- Uma possível solução seria validar o CEP antes de cotar.
- Outra solução seria colocar um validador com JS. Assim o cliente não consegue submeter a cotação, sem antes corrigir o CEP.

Enfim nada disso é muito complicado. Mas vai precisar de um Dev.

**Comment by gucalves** [comment]
@rafaelpatro, obrigado pelo retorno. Vou verificar esta questão do desenvolvimento, o problema é que são vários ceps e só vou saber qual não é "válido" quando o Correio rejeitar a postagem
